src/components/interaction/interaction-contoller.ts

```diff
@@ -0,0 +1,18 @@
+import Provider, { errors } from "oidc-provider";
+import { Request, Response } from "express";
+import { Config } from "../../config.js";
+
+export const interactionController =
+  (provider: Provider) => async (req: Request, res: Response) => {
+    const accountId = Config.getInstance().getSub();
+    const interactionDetails = await provider.interactionDetails(req, res);
+
+    if (interactionDetails.prompt.name === "login") {
```

Comment:
if prompt is none does this fail? as I think this is the case with most requests?

Reply:
Have pushed some changes to throw an `UnmetAuthenticationRequirement` error if the prompt is not login, looking at the authorisation handler, there might be some additional prompts I need to handle